Add CHANGELOG entry for 170e6c1 [ci-skip]

p8 opened the pull request:
For new appliciations `.attributes_for_inspect` is set to `[:id]` in
config/environment/production.rb.

In the console all the attributes are always shown.

### Checklist

Before submitting the PR make sure the following are checked:

* [x] This Pull Request is related to one change. Unrelated changes should be opened in separate PRs.
* [x] Commit message has a detailed description of what changed and why. If this PR fixes a related issue include it in the commit message. Ex: `[Fix #issue-number]`
* [x] Tests are added or updated if you fix a bug or add a feature.
* [x] CHANGELOG files are updated for the changed libraries if there is a behavior change or additional feature. Minor bug fixes and documentation changes should not be included.

skipkayhil commented on activerecord/CHANGELOG.md:
```diff
@@ -1001,7 +995,8 @@
 
-    In `development` and `test` mode, `attributes_for_inspect` will be set to `:all` by default.
+    By default `attributes_for_inspect` is set to `:all`.
+    For new appliciations it is set to `[:id]` in `config/environment/production.rb`.
```
Should the change in behavior be added to `Unreleased` at the top (since the current text is correct as of the release its a part of?)

p8 replied:
Ah yes, you are correct. I didn't know we had versioning in released CHANGELOGs.
I'll update it.

p8 committed:
Add CHANGELOG entry for 170e6c178b7358a87a7304bdf80273e0b153f6fd

For new appliciations `.attributes_for_inspect` is set to `[:id]` in
config/environment/production.rb.

In the console all the attributes are always shown.

skipkayhil reviewed (approved):
Perfect, thanks!
